home-manager/core.nix

```diff
@@ -36,32 +36,13 @@ in
         htop.enable = true;
         vim.enable = true;
         git.enable = true;
-        fish.enable = true;
-        fish.loginShellInit = "fish_add_path --move --prepend --path $HOME/.nix-profile/bin /run/wrappers/bin /etc/profiles/per-user/$USER/bin /run/current-system/sw/bin /nix/var/nix/profiles/default/bin";
+        #fish.enable = true;
+        #fish.loginShellInit = "fish_add_path --move --prepend --path $HOME/.nix-profile/bin /run/wrappers/bin /etc/profiles/per-user/$USER/bin /run/current-system/sw/bin /nix/var/nix/profiles/default/bin";
 
         home-manager.enable = true;
       };
 
       home.stateVersion = "23.05";
     };
-
-    systemd.services.wg0 = {
-      enable = true;
-
-      description = "wireguard interface for cross-node communication";
-      requires = [ "network-online.target" ];
-      after = [ "network-online.target" ];
-
-      serviceConfig = {
-        Type = "oneshot";
-      };
-
-      preStart = "${pkgs.wireguard-tools}/bin/wg-quick down /run/user/1000/wireguard/wg0.conf || true";
-      script = ''${pkgs.wireguard-tools}/bin/wg-quick \
-        up /run/user/1000/wireguard/wg0.conf
-      '';
-
-      wantedBy = [ "multi-user.target" ];
-    };
```

Comment:
I would rather revert this, because we want all hosts listed on this project to be connected into the intranetwork. We should run at least 1 aarch64-linux host, be it RPi4 or an Ampere VPS. Hence, unless a host has Wireguard connection, it cannot be part of the network, so making it optional doesn't make sense.

Reply:
This is true, let's revert this later before merging. I would rather not have this when I am testing the built images.